pkg/hubble/metrics/api/api.go

```diff
@@ -146,3 +128,101 @@ var registry = NewRegistry(
 func DefaultRegistry() *Registry {
 	return registry
 }
+
+// FlowFilters is a slice of filters to include or exclude flows.
+type FlowFilters []*pb.FlowFilter
+
+type ContextValues []string
+
+// ContextOptions is a structure used for configuring Hubble metrics context options
+type ContextOptionConfig struct {
```

Comment:
Could also name this `ContextOptions`? Since that's the name in the ConfigMap's yaml, and it looks like there used to be a type with similar name:
```go
Init(registry *prometheus.Registry, options Options)
```